src/com/company/Team.java

```diff
@@ -0,0 +1,70 @@
+package com.company;
+
+import java.util.ArrayList;
+
+public class Team {
+    private int teamId;
+    private String name;
+    private int noOfPlayers;
+    private int score=0;
+    private int notOut;
+    ArrayList<Player> member = new ArrayList<>();
+
+    public void setId(int id)
+    {
+        teamId = id;
```

Comment:
for setters, its a better practise to have the variable name in parameter same as field name in class. this would remove the hassle of creating a new variable name and easy to maintain what field you are exactly setting.
we can leverage "this" operator here. 
also getter/setter should have get/set followed by the field you are getting/setting. so this method should look like
public void setTeamId(int teamId) {
	this.teamId = teamId
}

Reply:
Thanks! Wasn't aware of "this". Implemented it.